templates/entry/comment/view.html.twig

```diff
@@ -46,11 +46,23 @@
         }) }}"
         data-controller="subject-list"
         data-action="{{- DYNAMIC_LISTS is same as V_TRUE ? autoAction : manualAction -}}">
-        {{ component('entry_comment', {
-            comment: comment.root ?? comment,
-            showEntryTitle: false,
-            showMagazineName: false,
-            showNested: true,
-        }) }}
+        {% set entryComment = comment.root ?? comment %}
+        {% if entryComment is defined and entryComment is not null %}
+            {{ component('entry_comment', {
+                comment: entryComment,
+                showEntryTitle: false,
+                showMagazineName: false,
+                showNested: true,
+            }) }}
+        {% else %}
+        <div class="section alert alert__danger">
+            <p>
+                <a href="{{ entry_url(entry) }}" class="btn btn-link btn__danger">
+                    <i class="fa-solid fa-arrow-left-long" title="{{ 'back'|trans }}"></i> {{ 'back'|trans }}
+                </a>
+                &nbsp; {{ 'comment_not_found'|trans }}
```

Comment:
What is the `&nbsp;` for?

Reply:
An additional space :)

Reply:
I know what it does, but why is it needed? 😅

Reply:
For a bit more space after the back button...

Reply:
I'd prefer it if you'd use margins for that

Reply:
I mean you can just inline style it in such situations imo

Reply:
can we just merge it ;P We have more important stuff haha

Reply:
Using inline CSS